deploy/operator.yaml

```diff
@@ -17,7 +17,7 @@ spec:
         - name: multiclusterhub-operator-pull-secret
       containers:
         - name: governance-policy-template-sync
-          image: quay.io/open-cluster-management/governance-policy-template-sync:latest-dev
+          image: quay.io/stolostron/governance-policy-template-sync:latest-dev
```

Comment:
latest-2.2?

Reply:
Updated. I'll update propagator also--I didn't update the tag there.